build/js/Game.js

```diff
@@ -93,12 +93,17 @@ Game.prototype.assignRandomColors = function() {
   var cardbacks = document.querySelectorAll('.card-back');
   var color;
```

Comment:
I don't see this variable being used anywhere, if its not then maybe take it out :)

Reply:
do you mean color = this.generateRandomHexcode(); ? That was removed.

Reply:
I see that now.  I accidentally just commented it out instead of deleting it, but it can be removed.